model/implementation/TestRunnerMessageService.php

```diff
@@ -36,6 +37,11 @@
  */
 class TestRunnerMessageService extends QtiRunnerMessageService
 {
+    /**
+     * @var string Controls whether a pause message is not sent on status, default false
+     */
+    private const FEATURE_FLAG_PROCTOR_NOT_LAUNCH_PAUSE_MESSAGE = 'FEATURE_FLAG_PROCTOR_NOT_LAUNCH_PAUSE_MESSAGE';
```

Comment:
Let's rename this to `FEATURE_FLAG_SKIP_PAUSED_ASSESSMENT_DIALOG`

Reply:
Good